apps/climatemappedafrica/payload.config.ts

```diff
@@ -0,0 +1,122 @@
+import path from "path";
+
+import { buildConfig } from "payload/config";
+import dotenv from "dotenv";
+import { cloudStorage } from "@payloadcms/plugin-cloud-storage";
+import { s3Adapter } from "@payloadcms/plugin-cloud-storage/s3";
+import { slateEditor } from "@payloadcms/richtext-slate";
+import { mongooseAdapter } from "@payloadcms/db-mongodb";
+import { webpackBundler } from "@payloadcms/bundler-webpack";
+import { CollectionConfig, GlobalConfig } from "payload/types";
+import nestedDocs from "@payloadcms/plugin-nested-docs";
+import seo from "@payloadcms/plugin-seo";
+import { sentry } from "@payloadcms/plugin-sentry";
+import { defaultLocale, locales } from "./src/payload/utils/locales";
+
+import Media from "./src/payload/collections/Media";
+import Pages from "./src/payload/collections/Pages";
+import Users from "./src/payload/collections/Users";
+
+dotenv.config();
+dotenv.config({ path: path.resolve(__dirname, "./.env.local") });
```

Comment:
I _think_ we no longer need to use `dotenv`... `next/env` should be able to do everything we need (check **RoboShield**)